doc/Installation/Installation-Ubuntu-1604-Apache.md

```diff
@@ -98,9 +98,17 @@ Edit the text which says `RANDOMSTRINGGOESHERE` and set your own community strin
     chmod +x /usr/bin/distro
     systemctl restart snmpd
 
-### Cron job
+### Background jobs
 
-    cp /opt/librenms/librenms.nonroot.cron /etc/cron.d/librenms
+If your upgrading LibreNMS, ensure to remove the crontab:
+
+    rm -fv /opt/librenms/librenms.nonroot.cron
```

Comment:
Why are you removing the source crontab?  

Should be: 

`rm -fv /etc/cron.d/librenms`

Reply:
Of course, my mistake. I fixed it.